l10n_nl_banks/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Odoo Experts
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Dutch banks",
+    "version": "10.0.1.0.0",
+    "author": "Odoo Experts, Onestein, Odoo Community Association (OCA)",
```

Comment:
Thanks for this, but Onestein didn't work on this module, as far as I know. Should be removed from here.